client/my-sites/email/email-providers-in-depth-comparison/index.tsx

```diff
@@ -13,8 +18,27 @@ type EmailProvidersInDepthComparisonProps = {
 };
 
 const EmailProvidersInDepthComparison: FunctionComponent< EmailProvidersInDepthComparisonProps > = () => {
+	const translate = useTranslate();
 	return (
-		<ComparisonTable emailProviders={ [ professionalEmailFeatures, googleWorkspaceFeatures ] } />
+		<>
+			<h1 className="email-providers-in-depth-comparison__header wp-brand-font">
+				{ translate( 'See how they compare' ) }{ ' ' }
```

Comment:
Why do we need that white space?

```suggestion
				{ translate( 'See how they compare' ) }
```

Can't we achieve the same thing with CSS?

Reply:
Ups, this seems to be the auto fix from PhpStorm, it sometimes adds that space when running the fixer.